header.go

```diff
@@ -113,18 +113,23 @@ func (h *ResponseHeader) SetContentRange(startPos, endPos, contentLength int) {
 //
 //   - If startPos is negative, then 'bytes=-startPos' value is set.
 //   - If endPos is negative, then 'bytes=startPos-' value is set.
-func (h *RequestHeader) SetByteRange(startPos, endPos int) {
+func (h *RequestHeader) SetByteRange(startPos, endPos []int) {
```

Comment:
This is a backwards compatible change we can not make.

Reply:
yeah, add  method of  `SetByteRanges`?